client/components/product-list-item.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class ProductListItem extends React.Component {
+  render() {
+    const price = (this.props.data.price / 100).toFixed(2);
+    return <>
+      <div className="col-4 d-flex justify-content-center">
+        <div className="card" style={{ width: '18rem' }}>
```

Comment:
It's best to keep styling in your css file and use a class, that way you can easily adjust later

Reply:
Ah okay, will be adjusted in the next update